packages/aws-cdk-lib/aws-stepfunctions-tasks/test/ecs/run-tasks-feature-flag.test.ts

```diff
@@ -0,0 +1,488 @@
+import { Template } from '../../../assertions';
+import * as autoscaling from '../../../aws-autoscaling';
+import * as ec2 from '../../../aws-ec2';
+import * as ecs from '../../../aws-ecs';
+import * as sfn from '../../../aws-stepfunctions';
+import { Stack } from '../../../core';
+import { ECS_REDUCE_RUN_TASK_PERMISSIONS } from '../../../cx-api';
+import * as tasks from '../../lib';
+
+let stack: Stack;
+let vpc: ec2.Vpc;
+let cluster: ecs.Cluster;
+
+/* eslint-disable quote-props */
```

Comment:
given how `run-tasks.test.ts` was implemented with the `beforeEach`, I think it makes more sense to test these in a new file.